### ISSUE TITLE
Can't frame grab on Yoda

### ISSUE THREAD

**Issue by lindakuhnz** [mbari-media-management/vars-annotation#108]
I’m still dead in the water for annotating :-(
Lonny has checked everything.

Begin forwarded message:

From: Linda Kuhnz <linda@mbari.org>
Subject: VARS can't framegrab and error messgae when opening tapeset
Date: February 7, 2019 at 9:37:05 AM PST
To: Brian Schlining <brian@mbari.org>
Cc: Lonny Lundsten <lonny@mbari.org>, Susan vonThun <svonthun@mbari.org>, Nancy Jacobsen-Stout <jana@mbari.org>

Restarted VARS, restarted computer, have frame grabbing apps open and set, preferences set correctly.
Pretty sure I have latest version (although not sure how I could tell).

Also getting an error message when opening a tape (see attached).

<img width="668" alt="screen shot 2019-02-07 at 9 30 52 am" src="https://user-images.githubusercontent.com/38045059/53030713-3daf3080-3420-11e9-9afc-c3a3cf3966f3.png">
<img width="428" alt="screen shot 2019-02-07 at 9 32 53 am" src="https://user-images.githubusercontent.com/38045059/53030717-40aa2100-3420-11e9-9411-631a97921af3.png">

**Comment by lindakuhnz** [comment]
Continued trouble shooting Yoda. Signed on as digital admin to see if I could framegrab on the computer under a different user; the attached screen grab shows that the computer is grabbing from Camtwist and not the Black Siphon feed as specified. Lonny: does us clue us in to something?

<img width="2002" alt="screen shot 2019-02-20 at 8 40 54 am" src="https://user-images.githubusercontent.com/38045059/53108716-5edd5300-34ec-11e9-9534-e23f73692d3f.png">

**Comment by lonnylundsten** [comment]
Its close to working. Looks like an issue with camtwist. I think its just a setting that got goofed up in camtwist or vars.

**Comment by lindakuhnz** [comment]
Lonny and I reviewed this situation again today. It's not CamTwist or the other apps causing the issue. We came to think that there is some kind of VARS-related preference file that is holding on to something that is causing the problem, evidence being that we can get everything to work on an alternate user account on Yoda (digitaladmin). I set up a brand new user for myself, re-installed everything and after getting all of the settings/preferences in VARS, Sharktopoda, Black Siphon, CamTwist, and Simple Client loaded I can now frame grab on Yoda from my new account using video tape or video files. Lonny may have deleted a preference file during the exercise as one thing I'm noticing is that VARS no longer defaults to my user name and my default tabs, etc. on VARS X (not a problem, but may be a hint at something happening behind the scenes)
